CHANGELOG.md

```diff
@@ -2,464 +2,311 @@
+  - Virtual DVD drives for templates
+- Containers: UI updates, refresh button
+- Foreman
+  - Added tag processing during provisioning
+  - Added inventory collection of direct and inherited host/host-group settings
+- Kubernetes 
+  - Inventory Collection
+  - EMS Refresh scheduling
+- The Foreman
+  - Provider refresh
+  - Enabled Reporting / Tagging
+  - Automate service models
+  - Zone enablement
+  - EMS Refresh scheduling
+- Cloud Providers
+ - Cloud Stacks: Summary and list views.
+ - Cloud images and instances: Added root device type to summary screens.
+ - Cloud flavors: Added block storage restriction to summary screens.
+ - Reporting
+- OpenStack
   - Inventory for Heat Stacks (Cloud and Infrastructure)
   - Connect Cloud provider to Infrastructure provider
-- Providers: General
-  - EMS Refresh scheduling for Foreman and Kubernetes
-- Rest API
-  - Providers CRUD
-  - Refresh via /api/providers
-- Provisioning
-  - Heat Orchestration provisioning through services
-  - Foreman
-     - Provisioning of bare metal systems
-     - Organization and location inventory
-     - Uses latest Foreman Apipie gem
-- Automate: Enhanced UI import
-- Fleecing
- - Qcow3 support
- - VSAN support (VMware)
- - OpenStack instances
-- UI
- - Foreman includes provisioning
+  - OpenStack Infrastructure Host Events
+  - Autoscale compute nodes
+  - Support for non-admin users to EMS Refresh
+  - Tenant relationships added to summary screens 
+- Amazon
+  - AWS Region EU Frankfurt
+  - Inventory collection for AWS CloudFormation
+  - Parsing of parameters from orchestration templates
+  - Providers: Amazon Events
+  - Enables event-based policies for AWS
+  - Added a backend attribute to identify public images.
+  - Added C4, D2, and G2 instance types.
+  - Virtualization type collected during EMS refresh for better
+    filtering of available types during provisioning.
+- Orchestration
  - Orchestration Stacks includes tagging
  - Orchestration templates
      - Create, edit, delete, tagging, ‘draft’ support
      - Can create a Service Dialog from template contents
- - HTML5 Consoles for OpenStack
-
-### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+21+Ending+Mar+30%2C+2015%22+label%3Aenhancement) 
-
-- Automate
- - Dynamic Dialog Date/Time Control
- - Disabled domains clearly marked in UI
- - Automate entry point selection reduced to state machine classes
-- IPv6
- - More code paths communicate via hostname
- - URI building with IPv6 safety
-- UI: tagging support and toolbars on list views
-
-### [Fixed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+21+Ending+Mar+30%2C+2015%22+label%3Abug)
-
-- Ruby2 trap logging and worker row status
-- Update STMP domain length to match SMTP host length
-
-## Unreleased - as of Sprint 20 end 2015-03-09
-
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+20+Ending+Mar+9%2C+2015%22+label%3Aenhancement)
+ - Provisioning dialog generator
```

Comment:
Delete.  This is the same thing as "Can create a Service Dialog from template contents".